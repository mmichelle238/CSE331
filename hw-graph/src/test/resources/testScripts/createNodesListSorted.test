# create a graph and 3 nodes. make sure
# that the list of the nodes is sorted alphabetically

CreateGraph A
AddNode A n3
AddNode A n2
AddNode A n1
ListNodes A