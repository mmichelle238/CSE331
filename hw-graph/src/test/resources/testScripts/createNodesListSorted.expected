# create a graph and 3 nodes. make sure
# that the list of the nodes is sorted alphabetically

created graph A
added node n3 to A
added node n2 to A
added node n1 to A
A contains: n1 n2 n3